Lib9c.Models/Items/OrderLock.cs

```diff
@@ -0,0 +1,35 @@
+using Bencodex.Types;
+using Lib9c.Models.Exceptions;
+using Nekoyume.Model.State;
+using ValueKind = Bencodex.Types.ValueKind;
+
+namespace Lib9c.Models.Items;
+
+/// <summary>
+/// <see cref="Nekoyume.Model.Item.OrderLock"/>
+/// </summary>
+public record OrderLock : Lock
+{
+    public Guid OrderId { get; init; }
+
+    public override IValue Bencoded => new List(
+        base.Bencoded,
+        OrderId.Serialize());
```

Comment:
It seems different with the `OrderLock.Bencoded` 🤔

Reply:
I fixed it!